pages/playground.js

```diff
@@ -87,6 +88,22 @@ export default function Playground() {
     }
   };
 
+  const buttonProps = () => {
+    if (!formatComplete) {
+      return { text: "Format", onClick: handleFormatJson, primary: false };
+    } else if (formatComplete && !validateComplete) {

```

Comment:
Because of the `return` early in the condition above these don't need to be `else if`, you can see my example here https://github.com/EddieHubCommunity/LinkFree/pull/7141#pullrequestreview-1444838449

Reply:
Thanks,
I'll keep this in mind for further
Made changes as you said